### ISSUE TITLE
Handle insufficient_quota error

### ISSUE THREAD

**Issue by soderluk** [mattvr/ShellGPT#23]
If your `OPENAI_API_KEY` exceeds the quota, you'll only get an empty response from `gpt`.
The actual response from openai:
```
{
    "error": {
        "message": "You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.",
        "type": "insufficient_quota",
        "param": null,
        "code": "insufficient_quota"
    }
}
```
This error should be handled, and a message should be shown to the user that their quota has been exceeded. `lib/ai.ts:258` is where the result is decoded into this message.

**Comment by mattvr** [comment]
Thanks @soderluk for debugging, just fixed in [0.3.9](https://github.com/mattvr/ShellGPT/releases/tag/0.3.9)

**Comment by mattvr** [comment]
Thanks again @soderluk

I tried to address again in the latest version [0.4.0](https://github.com/mattvr/ShellGPT/releases/tag/0.4.0). I'm not able to reproduce this specific error, so lmk if it's working now.
